app/src/main/java/com/invisibleteam/goinvisible/util/TagUtil.java

```diff
@@ -0,0 +1,144 @@
+package com.invisibleteam.goinvisible.util;
+
+
+import android.support.annotation.VisibleForTesting;
+import android.text.TextUtils;
+
+import java.math.BigDecimal;
+
+import javax.annotation.Nullable;
+
+public class TagUtil {
+
+    private static final String GEO_MINUTES_CHAR = "\'";
+    private static final String GEO_SECONDS_CHAR = "\'\'";
+    private static final String DEFAULT_DENOMINATOR = "/1,";
+    private static final String DENOMINATOR_CHAR = "/";
+    private static final int GEO_SECONDS_DENOMINATOR_VALUE = 1000000;
+
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    static final char DEGREE_CHAR = (char) 0x00B0;
+    public static final int SIX_DECIMAL_NUMBER_SCALE = 6;
+
+    public static String parseDoubleGPSToGeoGPS(double value) {
+        final int[] geoGPS = convertDoubleGPSValueToGeoGPSValue(value);
+
+        int degrees = geoGPS[0];
+        int minutes = geoGPS[1];
+        int seconds = geoGPS[2];
+
+        return new StringBuilder()
```

Comment:
Some parts of this builder are duplicated in the second method but I don't think that we should expose method for that.